### DISCUSSION TITLE
SSLSocket closes socket both socket endpoints on a SocketTimeoutException

### DISCUSSION THREAD

**Discussion by rgupta-git** [adoptium/adoptium-support#293] [Q&A] [answered]
Hi Guys
Any idea in which OpenJDK-8 release, below defect will be backported?

https://bugs.openjdk.java.net/browse/JDK-8261783

Thanks
Ravi S Gupta

**Comment by karianna** [comment, marked as answer]
Looks like it's been assigned but won't be in 8u292, so probably 8u302 at the earliest.

**Reply by rgupta-git** [reply]
Many thanks @karianna, this defect was assigned on same day it created "2021-02-15", however **STATUS** is still **NEW**, hope this will be picked up for upcoming release?

Can you suggest any workaround here?

**Reply by karianna** [reply]
Looks like it's been assigned so hopefully, it'll make it in soon.  I'm afraid I can't think of an easy workaround.
